src/wp-admin/setup-config.php

```diff
@@ -176,7 +176,7 @@
 		echo '		<td><input name="prefix" id="prefix" type="text" value="cp_" size="25" /> ' .
 			sprintf(
 				'<a href="%s" target="_blank" rel="noopener">' . __( 'Learn More' ) . '</a>',
-				esc_url('https://docs.classicpress.net/installing-classicpress/#installation-steps')
+				esc_url('https://link.classicpress.net/installing-classicpress/#installation-steps')
```

Comment:
... and this one could be https://link.classicpress.net/installing-classicpress/installation-steps I guess, leaving room for other `installing-classicpress/*` links in the future.